libpod/pod_api.go

```diff
@@ -581,20 +581,22 @@ func (p *Pod) Status() (map[string]define.ContainerStatus, error) {
 }
 
 func containerStatusFromContainers(allCtrs []*Container) (map[string]define.ContainerStatus, error) {
-	// We need to lock all the containers
```

Comment:
Of course, I saw this comment and my assumption is this is not correct now. Please let me know if I am wrong.